dashboard/app/models/user.rb

```diff
@@ -161,7 +161,9 @@ class User < ActiveRecord::Base
 
   SYSTEM_DELETED_USERNAME = 'sys_deleted'
 
+  # constants for resetting user secret words/picture
```

Comment:
super teeny tiny nit: [Style Guide](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md) says we prefer capitalization in comments.